### PR TITLE
chapter 11

### PR DESCRIPTION
**Pull request by daniadore**
Ready

### CODE REVIEW

**Comment by daniadore** [comment]
Ready

File: app/assets/stylesheets/account_activations.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the AccountActivations controller here.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/account_activations.scss:1]: delete it because not use

File: app/helpers/account_activations_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module AccountActivationsHelper
```
Comment (ngocvt-0484) [inline app/helpers/account_activations_helper.rb:1]: delete this file

File: test/mailers/user_mailer_test.rb [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+class UserMailerTest < ActionMailer::TestCase
+  test "account_activation" do
+    mail = UserMailer.account_activation
+    assert_equal "Account activation", mail.subject
+    assert_equal ["to@example.org"], mail.to
+    assert_equal ["from@example.com"], mail.from
+    assert_match "Hi", mail.body.encoded
+  end
+
+  test "password_reset" do
```
Comment (ngocvt-0484) [inline test/mailers/user_mailer_test.rb:12]: dont need do test
so delete all file on test folder

**Comment by daniadore** [comment]
Ready

File: test/controllers/account_activations_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class AccountActivationsControllerTest < ActionDispatch::IntegrationTest
```
Comment (anlvq-1058) [inline test/controllers/account_activations_controller_test.rb:3]: please delete this file

File: test/controllers/static_pages_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```
Comment (anlvq-1058) [inline test/controllers/static_pages_controller_test.rb:1]: please delete this file

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by daniadore** [comment]
Ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**